space-plugins/nuxt-base/app.config.ts

```diff
@@ -1,4 +1,12 @@
+import type { AppBridgeConfig } from './types/appBridge';
+
 export default defineAppConfig({
+	appBridge: {
+		type: 'space-plugin',
+		enabled: false,
+		oauth: true,
+		origin: 'https://app.storyblok.com',
```

Comment:
can the `origin` be something else?

Reply:
i mean the value, if its always https://app.storyblok.com or can be something else as well.

Reply:
That makes total sense, haven't thought about that. Thank you for the info 🙏